diff --git a/libs/lottie.js b/libs/lottie.js
index 9ade8c1378595d3d06077fd9f701ed8a5837f0b3..d3efeb505809a869488796cb8cd277e1a4980d23 100644
--- a/libs/lottie.js
+++ b/libs/lottie.js
@@ -1044,15 +1044,7 @@ const lottie = (() => {
     };
   }();
   const ImagePreloader = function() {
-    var proxyImage = function() {
-      var canvas = createTag("canvas");
-      canvas.width = 1;
-      canvas.height = 1;
-      var ctx = canvas.getContext("2d");
-      ctx.fillStyle = "rgba(0,0,0,0)";
-      ctx.fillRect(0, 0, 1, 1);
-      return canvas;
-    }();
+    var proxyImage
     function imageLoaded() {
       this.loadedAssets += 1;
       if (this.loadedAssets === this.totalImages && this.loadedFootagesCount === this.totalFootages) {
@@ -1218,6 +1210,15 @@ const lottie = (() => {
       }
     }
     function ImagePreloaderFactory() {
+      proxyImage = function() {
+        var canvas = createTag("canvas");
+        canvas.width = 1;
+        canvas.height = 1;
+        var ctx = canvas.getContext("2d");
+        ctx.fillStyle = "rgba(0,0,0,0)";
+        ctx.fillRect(0, 0, 1, 1);
+        return canvas;
+      }(); 
       this._imageLoaded = imageLoaded.bind(this);
       this._footageLoaded = footageLoaded.bind(this);
       this.testImageLoaded = testImageLoaded.bind(this);
